core/templates/dev/head/pages/exploration_editor/issues_tab/IssuesTab.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the exploration issues tab in the exploration
+ * editor.
+ */
+
+oppia.controller('IssuesTab', ['$scope', function($scope) {
+  $scope.resolvedIssueCount = 13;
+  $scope.archivedIssueCount = 30;
+
+  var issueTypeMaterialIconMapping = {
+    'Feedback': 'feedback',
+    'Playthrough': 'gamepad',
+    'Unresolved Answer': 'help',
+  };
+
+  $scope.issueTypeToMaterialIcon = function(issueType) {
+    if (issueTypeMaterialIconMapping.hasOwnProperty(issueType)) {
+      return issueTypeMaterialIconMapping[issueType];
+    } else {
+      return 'bug_report';
```

Comment:
Is defaulting to bug_report always correct? Can we somehow ensure ``issueType`` is always correct? If we can, then this should be an error rather than a default.

Reply:
I considered it, but this feels too trivial to show a user-facing error; maybe rendering "nothing" is good enough?

Reply:
Marking as resolved for now, let me know if you feel strongly against this.